Formula/elastix.rb

```diff
@@ -0,0 +1,15 @@
+class Elastix < Formula
+  desc "Toolbox for rigid and nonrigid registration of images"
+  homepage "http://elastix.isi.uu.nl/index.php"
+  url "http://elastix.isi.uu.nl/download/elastix_macosx64_v4.8.zip"
```

Comment:
We'll want to build this from source.

Reply:
As you said, probably a better fit for Cask.